components/Form/AddFlightForm.js

```diff
@@ -147,10 +150,10 @@ export default function AddFlightForm() {
 
   const router = useRouter();
```

Comment:
Delete the entire block of code that is commented out. Don't keep duplicate code around.